lib/features/Auth/login/data/datasources/shared_prefs.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+// This class contains the constants - Email id Key and Password key
+class Constant{
+final String key_Email="email";
```

Comment:
Name non-constant identifiers using lowerCamelCase
```suggestion
final String keyEmail = "email";
```

Reply:
use `static const` instead of `final`
```suggestion
static const String keyEmail = "email";
```